www/js/lib/uiUtil.js

```diff
@@ -84,12 +84,45 @@ define([], function() {
         return url.replace(regexpRemoveUrlParameters, "$1");
     }
 
+    /**
+     * Displays a Bootstrap warning alert with information about how to access content in a ZIM with unsupported active UI
+     */
+    function displayActiveContentWarning() {
+        // We have to add the alert box in code, because Bootstrap removes it completely from the DOM when the user dismisses it
```

Comment:
Could you give me some more explanation?
What was the issue?

Reply:
@mossroy PS from the Bootstrap API linked above:

> Note that closing an alert will remove it from the DOM.

Reply:
I think we should investigate to find a better way to do that.
It should be avoided to have big chunks of HTML in a javascript file.

A few ideas : 
- the alert DOM element could be hidden somewhere else, and copied when we need to display it. So that bootstrap only removes the copy when the user dismisses it
- or, as you were mentioning, we could override the behavior of the X, like in https://stackoverflow.com/questions/20494887/my-bootstrap-alert-will-not-display-after-being-closed or in https://stackoverflow.com/questions/13550477/twitter-bootstrap-alert-message-close-and-open-again

But there might be better ideas.

Anyway, if it takes too much time, this might be left as is for now, provided that it will be fixed in next version (through a new issue)

Reply:
I have created #470 to deal with this issue. I'm wary about overriding Bootstrap standard behaviour, so would prefer the copy option, but we shall see.